package_policies/M10.md

```diff
@@ -1,4 +1,5 @@
-**M10.** It is mandatory that each package have an xSDK-accessible **repository** (not necessarily
-publicly available), where the development version of the package is available. Support for taking **pull
-requests** is also recommended since this will greatly ease the effort among the development teams to
-improve interoperability and quickly provide bug fixes.
+**M10.** It is mandatory that each package have a **public repository**, for example at GitHub,
+GitLab or Bitbucket, where the development version of the package is available. Support for taking
+**pull requests** is also recommended since this will ease the efforts among xSDK library
+development teams to improve interoperability and quickly provide bug fixes and also allow users
+to provide improvements to the packages.
```

Comment:
```suggestion
GitLab or Bitbucket, where the development version of the package is available. Recommend
supporting **pull requests**,  this enables xSDK library developers  and users
to provide improvements, bug fixes and interoperability fixes. 
```

Reply:
Either seems fine. If Satish's, I would edit slightly, replacing "this" with "which", to avoid the comma splice:
"... supporting **pull requests**, which enables ..."

Reply:
Thanks! I edited/updated my suggested change.